src/components/activities/pharmacyActivity/pharmaceuticalStock/DischargeMovement.tsx

```diff
@@ -0,0 +1,136 @@
+    (state) => state.pharmacy.dischargeMovements.error?.message
+  ) as string;
+
+  const { breadcrumbMap, setBreadcrumbMap } = useOutletContext<{
+    breadcrumbMap: Record<string, string>;
+    setBreadcrumbMap: (map: Record<string, string | undefined>) => void;
+  }>();
+
+  const addBreadcrumb = () => {
+    setBreadcrumbMap({
+      ...breadcrumbMap,
+      [t("pharmacy.labels.pharmaceutical-stock")]:
+        PATHS.pharmacy_pharmaceuticalstock,
+      [t("pharmacy.labels.DisCharge-movement")]:
+        PATHS.pharmacy_pharmaceuticalstock_discharge,
+    });
+  };
+
+  const removeBreadcrumb = () => {
+    setBreadcrumbMap({
+      ...breadcrumbMap,
+      [t("pharmacy.labels.pharmaceutical-stock")]: undefined,
+      [t("pharmacy.labels.DisCharge-movement")]: undefined,
+    });
+  };
+
+  const handleSubmit = (values: MovementDTO[]) => {
+    console.log("handleSubmit", values);
+    dispatch(dischargeMovements({ ref: "REF123", movementDTO: values }));
+  };
+
+  useEffect(() => {
+    if (activityTransitionState === "TO_RESET") {
+      dispatch(resetDischargeMovements());
+      setActivityTransitionState("IDLE");
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [dispatch, activityTransitionState]);
+
+  useEffect(() => {
+    addBreadcrumb();
+    return () => {
+      const newMap = { ...breadcrumbMap };
+      delete newMap[t("pharmacy.labels.pharmaceutical-stock")];
+      delete newMap[t("pharmacy.labels.DisCharge-movement")];
+      setBreadcrumbMap(newMap);
+    };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  useEffect(() => {
+    dispatch(getMedicals());
+  }, [dispatch]);
+
+  return (
+    <PharmacyActivityContent
+      data-cy="DisCharge-movement"
+      title={t("pharmacy.labels.DisCharge-movement")}
+    >
+      <div className="DisCharge-movement">
+        <DisChargeMovementForm
+          onSubmit={handleSubmit}
+          onCancel={() => {
+            const newMap = { ...breadcrumbMap };
+            delete newMap[t("pharmacy.labels.pharmaceutical-stock")];
+            delete newMap[t("pharmacy.labels.DisCharge-movement")];
+            setBreadcrumbMap(newMap);
+            navigate(PATHS.pharmacy_pharmaceuticalstock, { replace: true });
+          }}
+        />
+      </div>
+      <ConfirmationDialog
+        isOpen={createStatus === "SUCCESS"}
+        title="Discharge movement created"
+        icon={checkIcon}
+        info="Discharge movement created successfully"
+        primaryButtonLabel="Ok"
+        handlePrimaryButtonClick={() => {
+          const newMap = { ...breadcrumbMap };
+          delete newMap[t("pharmacy.labels.pharmaceutical-stock")];
+          delete newMap[t("pharmacy.labels.DisCharge-movement")];
+          dispatch(resetDischargeMovements());
+          setBreadcrumbMap(newMap);
+          setActivityTransitionState("TO_RESET");
+          navigate(PATHS.pharmacy_pharmaceuticalstock, { replace: true });
+        }}
```

Comment:
```suggestion
        handlePrimaryButtonClick={handleReset}
```
const handleReset = useCallback(...)